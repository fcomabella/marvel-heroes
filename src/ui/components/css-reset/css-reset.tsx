import { css, Global } from '@emotion/react';
import { ReactNode } from '@tanstack/react-router';
import emotionReset from 'emotion-reset';

export const CssReset = (): ReactNode => (
  <Global
    styles={css`
      ${emotionReset}

      *, *::after, *::before {
        box-sizing: border-box;
        -moz-osx-font-smoothing: grayscale;
        -webkit-font-smoothing: antialiased;
        font-smoothing: antialiased;
      }

      :root {
        font-family: 'Roboto Condensed', sans-serif;
      }
    `}
  />
);
