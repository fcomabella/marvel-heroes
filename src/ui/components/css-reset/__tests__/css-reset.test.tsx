import { render } from '@testing-library/react';
import { CssReset } from '../css-reset';

describe('CssReset component', () => {
  it('Should render', () => {
    const { container } = render(<CssReset />);
    expect(container).toMatchSnapshot();
  });
});
