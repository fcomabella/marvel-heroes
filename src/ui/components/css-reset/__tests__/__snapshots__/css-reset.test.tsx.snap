// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CssReset component > Should render 1`] = `<div />`;
